xgbgen/request_reply.go

```diff
@@ -27,10 +27,6 @@ func (rs Requests) Less(i, j int) bool { return rs[i].xmlName < rs[j].xmlName }
 // It also initializes the reply if one exists, and all fields in this request.
 func (r *Request) Initialize(p *Protocol) {
 	r.srcName = SrcName(p, r.xmlName)
-	if p.isExt() {
-		r.srcName = r.srcName
```

Comment:
I suspect the logic here was meant to perform some sort of override of `r.srcName`, but was typo’ed into being a useless assignment. Like, `r.srcName = p.ExtName`

Consider what could have been intended here, rather than just removing the code.